api/routes.ts

```diff
@@ -21,5 +23,7 @@ router.use('/userList', getUserList)
 router.use('/userDetails', getUserList)
 router.use('/jurisdictions', getJurisdictions)
 router.use('/payments/:account', payments)
+router.use('/user/:userId/suspend', suspendUser)
```

Comment:
Not sure we should have parameterised routes in the main route file, can we not split into `router.use('/user', userRouter)` or the like?

Reply:
@paddybasi  Again I would not ideally like to touch this code as as I said above. We need to remember that this is not 1 feature. Many features which were added are now being added to develop from demo. Hence, I would prefer not to change any code until it's a sure stopper. Thanks.